.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+trailingComma: "none",
+semi: false,
```

Comment:
What is it?

Reply:
i created this prettier to be one script entry point
like test

Reply:
it's right doing this? this **.prettierrc** it has other rules, like global rules that **.eslintrc** it'll get also the **.prettier**("_eslint:recommended_" & _.prettier_ will run as extends )

Reply:
I checked prettier, and it's optional,  there are others rules, but if you want to add , it's up to you